src/test/java/tools/jackson/dataformat/xml/node/JsonNodeBasicDeserTest.java

```diff
@@ -43,7 +44,7 @@ public void testMixedContent() throws Exception
     {
         JsonNode fromXml = XML_MAPPER.readTree("<root>first<a>123</a>second<b>abc</b>last</root>");
         final ObjectNode exp = XML_MAPPER.createObjectNode();
-        exp.putArray("")
+        exp.putArray(FromXmlParser.DEFAULT_TEXT_PROPERTY)
```

Comment:
Ohhhhhhh. This is NOT good -- what used to be something like:

```
{
  "":["first","second","last"],
  "a":"123",
  "b":"abc"
}
```

now looks like:

```
{
  "<xml:text>":["first","second","last"],
  "a":"123",
  "b":"abc"
}
```

Reply:
> We need to figure out another way to handle the issue, I think.

I mean this PR was just a naive shot at solving #306 by changing the property name, if it has to be `""` I'm fine with closing this PR - Because I do not know how to make it so, that the property name differns from the nominal key.

Reply:
No Problem, thanks for having a look at it. I will close this for now. Feel free to reopen or use the code in any way, shape or form you see fit.

Reply:
Ok. Thank you for your help here @duoduobingbing .